### DISCUSSION TITLE
Session commands bound to aliases don't seem to work

### DISCUSSION THREAD

**Discussion by stormdragon2976** [scandum/tintin#102] [Bugs]
Adding something like the following:

    #alias {^+$} {#session {+}}

Doesn't actual switch to the next session if you type +.

**Comment by scandum** [comment]
Maybe try:
```
#alias {^+$} {#line verbose #session {+}}
```
So you can see what is going on.

**Reply by stormdragon2976** [reply]
I think that was the whole problem, it wasn't giving the message to let me know something had happened. It seems to be working just fine now, thank you. :)
